### ISSUE TITLE
Sprint 6 Review

### ISSUE THREAD

**Issue by ksheedy** [the-spice-mice/web-app#89]

**Comment by ksheedy** [comment]
This week went very smoothly. I was assigned to create a pattern that uses tabs to navigate around a page. This included using javascript, which I am not terribly comfortable using. I was able to follow a pattern that Adam made last year as an example for the pattern, which was very helpful. I was able to use the example from Adam, but simplify it for our specific needs.

**Comment by hollixholton** [comment]
During this sprint, I finalized the banner photo pattern and coded the pattern for page numbers. For the most part, it seemed to go pretty smoothly. I found some good tutorials to teach me how to structure the code and found out that the pattern I was coding is actually called a "Pagination". I'm not sure how it's pronounced but it sounds pretty funny in my head. 

I did, however, run into some issues with overrides in the CSS. I'm learning that I need to be a lot more specific with my CSS selectors. 

Adam and Katie helped tremendously, as per ush.
